### ISSUE TITLE
Missing css for DateTimePicker

### ISSUE THREAD

**Issue by NicolaIsotta** [TheCoder4eu/BootsFaces-OSP#505]
I get this error when I have a dateTimePicker in the page:
![error](https://cloud.githubusercontent.com/assets/18428155/18478785/c66f89b6-79d2-11e6-97d6-9b3befc169a0.jpg)
I have this in web.xml:

``` xml
    <context-param>
        <param-name>BootsFaces_THEME</param-name>
        <param-value>custom</param-value>
    </context-param>
```

In fact, the css file is not in the jar:
![jar_content](https://cloud.githubusercontent.com/assets/18428155/18478922/4f8889dc-79d3-11e6-90f6-b9c3dff9fb83.jpg)

**Comment by stephanrauh** [comment]
@zhedar Can you please have a look at this bug? IMHO, everything is OK: the resource is added correctly, and it shouldn't be in the `other` folder, which is correct, too. Or should it be in the `other` folder because it's a non-standard resource file?

**Comment by zhedar** [comment]
As you said the dateTimePicker isn't a standard component, so we may need to discuss, if this is a bug or works as intended. If you want to provide your own css files, you may as well do so.

@NicolaIsotta Also, if you want a quick workaround you could try to [embed the file yourself](https://github.com/smalot/bootstrap-datetimepicker/blob/master/css/bootstrap-datetimepicker.min.css) into the bsf/css/other directory.

After a quick lookup I think we should include the file though, since we do the same with components like bootstrap-tree. I guess we only need to fix that in the gradle build.

Otherwise we may want to put all these theme-unspecific stylesheets into an own library, so we don't need to replicate them for every theme. I recently built and used a custom bootsfaces-jar with only the default themes. The difference in jar size was ~1.4MB vs ~>4MB.

**Comment by TheCoder4eu** [comment]
This issue is solved in v1.0.0
